src/utils/hederaUtils.js

```diff
@@ -240,6 +227,28 @@ module.exports = class HederaUtils {
     await this.generateED25519(client, num, startup, logger);
   }
 
+  static async createAccount(client, wallet, accountNum, logger, startup) {
+    const tx = await new HederaSDK.AccountCreateTransaction()
+      .setKey(
+        HederaSDK.PublicKey.fromString(
+          wallet._signingKey().compressedPublicKey
+        )
+      )
+      .setInitialBalance(HederaSDK.Hbar.fromTinybars(10000000000000))
```

Comment:
could the starting balances for all 3 account types be set to 10m HBAR as per the request in #291 ?

Reply:
Let's make this a configureable value that can be set on startup.
@georgi-l95 if it's a quick fix let's add here, if not follow up PR :)

Reply:
I've added option `--balance` with which you can set starting hbar balance. Currently by default, if you don't set this options is 10000 hbar. (same as now)